### DISCUSSION TITLE
Documentation screenshot for "Body with multiple examples" incorrect

### DISCUSSION THREAD

**Discussion by ndevenish** [fastapi/fastapi#10054] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Annotated

from fastapi import Body, FastAPI
from pydantic import BaseModel

app = FastAPI()


class Item(BaseModel):
    name: str
    description: str | None = None
    price: float
    tax: float | None = None


@app.put("/items/{item_id}")
async def update_item(
    *,
    item_id: int,
    item: Annotated[
        Item,
        Body(
            examples=[
                {
                    "name": "Foo",
                    "description": "A very nice Item",
                    "price": 35.4,
                    "tax": 3.2,
                },
                {
                    "name": "Bar",
                    "price": "35.4",
                },
                {
                    "name": "Baz",
                    "price": "thirty five point four",
                },
            ],
        ),
    ],
):
    results = {"item_id": item_id, "item": item}
    return results
```

### Description

https://fastapi.tiangolo.com/tutorial/schema-extra-example/#body-with-multiple-examples has the new "Body.examples is a list" form. 

But the screenshot in [Examples in the Docs UI](https://fastapi.tiangolo.com/tutorial/schema-extra-example/#examples-in-the-docs-ui) underneath is from the old "Dictionary with keys as the example name form". - it shows example names, which have been removed from the code sample, and shows the ability to select multiple examples, which is currently broken.

It currently actually renders identically to the screenshot earlier in the documentation, without multiple examples:
![image](https://github.com/tiangolo/fastapi/assets/529963/a8b3e63a-53df-45c2-acaa-e82ccf5937c4)

### Operating System

macOS

### FastAPI Version

0.101.0

### Pydantic Version

2.1.1

### Python Version

3.11.4

### Additional Context

This is **not** a direct duplicate of https://github.com/tiangolo/fastapi/discussions/9794. That issue is trying to get multiple examples working (which I haven't worked out how to do yet). 

This is specifically raising that the screenshot and documentation is out-of-date. 

A comment on https://github.com/tiangolo/fastapi/discussions/9794 did notice this issue: https://github.com/tiangolo/fastapi/discussions/9794#discussioncomment-6395706 - but that seems to have been missed as it came after the answer was marked. It's possible that https://github.com/tiangolo/fastapi/pull/9866 might fix the issue, but the PR does not seem to have updated the documentation to match the new PR implementation either.

In the meantime, the documentation is somewhat misleading with unachievable screenshots of the docs.

**Comment by moskalen** [comment]
I have the same issue when only the first example from the list is visible in `/docs` for 

**Python Version**

3.10.10

(all other details are the same)

**Comment by ycc140** [comment]
I also have the  same issue on Windows.

**Comment by tiangolo** [comment, marked as answer]
Yep, it's because Swagger UI still doesn't support multiple JSON Schema examples...

But there's a new parameter `openapi_examples` just for this, to allow declaring examples that show up in Swagger UI while it implements support for multiple JSON Schema examples. It's available now in FastAPI 0.103.0, the new docs are here: https://fastapi.tiangolo.com/tutorial/schema-extra-example/#openapi-specific-examples 🎉

**Reply by ndevenish** [reply]
Excellent! Simple to migrate from pre-100, and the documentation is updated too! Thanks!
